---
title: "RAD-Seq"
author: "Kalin Nonchev, Silvan Vollmer, Tudor Jumuga"
date: "9/28/2021"
output: html_document
---

### Summary RAD-Seq

Restriction site-associated DNA sequencing (RAD-Seq) is a method that provides 
genomic data with high resolution for any organism at fair price by reducing the 
complexity of the target genome across many individuals. \
RAD-Seq allows the identification and the scoring of many genetic markers 
(microsatellites, single nucleotide polymorphisms, insertionâ€“deletion polymorphisms, 
etc) over a group of samples by preprocessing the raw DNA material and then utilizing 
the Illumina technology underneath. The preprocessing step is based on the use of 
restriction enzymes to cut the desired fragments from the DNA and on the use of 
molecular identifiers to pinpoint the particular individual and their sequencing 
reads.

### Link

| Technology        | Application | Statistics|        
| ------------- |:-------------:| :-------------:| 
| RAD-Seq      |  (fractional) genome sequencing | TMM normalization, pairwise exact tests of negative binomial distribution, Benjamini-Hochberg adjusted significance levels (to control FDR), Average Linkage Clustering, PCA, Smoothing Splines, Fisher's exact test (Schield et al.)|

#### References

* Baird, N.A., Etter, P.D., Atwood, T.S., Currey, M.C., Shiver, A.L., Lewis, Z.A., Selker, E.U., Cresko, W.A. and Johnson, E.A., 2008. Rapid SNP discovery and genetic mapping using sequenced RAD markers. PloS one, 3(10), p.e3376.
* Davey, J.W. and Blaxter, M.L., 2010. RADSeq: next-generation population genetics. Briefings in functional genomics, 9(5-6), pp.416-423.
* Schield, D.R., Walsh, M.R., Card, D.C., Andrew, A.L., Adams, R.H. and Castoe, T.A., 2016. EpiRADseq: scalable analysis of genomewide patterns of
methylation using next-generation sequencing, Methods in Ecology and Evolution, 7, p.63.



#### GitHub Usernames 

* KalinNonchev
* svollm
* tudorandreij
